cstool/Makefile

```diff
@@ -37,6 +37,7 @@ endif
 
 clean:
 	${RM} -rf *.o $(TARGET)
+	${RM} -rf *.d
```

Comment:
there is no more .o files?

Reply:
Yes, it just creates .d files aside to the .o files.